test/printingTools.js

```diff
@@ -0,0 +1,409 @@
+// This test can be run directly using mocha:
+// $ npx mocha test/printingTools.js
+// Using truffle test works but it's much slower, since it needs to compile contracts and deploy them to the blockchain.
+
+const assert = require("assert")
+const BN = require("bn.js")
+
+const { toErc20Units, fromErc20Units, bnMaxUint, bnOne } = require("../scripts/utils/printing_tools")
+
+const goodTwoWayPairs = [
```

Comment:
This is the only test sample arranged outside of the actual unit test. Can you clean this?

Reply:
This array is used by both `describe`s, so I'm not sure how to to include it in a unit test without duplicating the code.